generator/whitelist.ts

```diff
@@ -338,6 +338,10 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'sql/resource-manager',
         namespace: 'Microsoft.Sql',
     },
+    { 
+        basePath: 'storsimple8000series/resource-manager',
+        namespace: 'Microsoft.StorSimple',
+    },
```

Comment:
Because there are two separate directories for the same RP (storsimple8000series and storsimple1200series), please could you add a suffix here to avoid the files overwriting each other:
`suffix: '8000',`